presto-docs/src/main/sphinx/connector/cassandra.rst

```diff
@@ -196,3 +207,8 @@ This table can be described in Presto::
 This table can then be queried in Presto::
 
     SELECT * FROM cassandra.mykeyspace.users;
+
+When using SSL, use both the cassandra.ssl properties outlined above, as well as the followinf thrift port config:
```

Comment:
Oh, also, we just removed Thrift from the Cassandra connector, since they were deprecated and don't work by default with the newest version of Cassandra: https://github.com/prestodb/presto/pull/7013
So you can remove this section in the docs.